client/src/entrypoints/admin/comments.js

```diff
@@ -1,7 +1,268 @@
-import comments from '../../components/CommentApp/comments';
+import { initCommentApp } from '../../components/CommentApp/main';
+import { STRINGS } from '../../config/wagtailConfig';
 
 /**
  * Entry point loaded when the comments system is in use.
  */
-// Expose module as a global.
-window.comments = comments;
+window.comments = (() => {
```

Comment:
I've taken the contents of comments.js and pasted it here. Wrapped it in ``window.comments = (() => { ... })()`` and changed exports to a return value.